docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+#### push_worker_count_high_priority
+
+- **Default**: 3
+- **Unit**: N/A
+- **Description**: The number of threads used to handle a load task with HIGH priority.
+
+#### transaction_publish_version_worker_count
+
+- **Default**: 0
+- **Unit**: N/A
+- **Description**: The maximum number of threads used to publish a version. When this value is set to less than or equal to 0, the system uses half of the CPU core count as the value, so as to avoid insufficient thread resources when import concurrency is high but only a fixed number of threads are used. From v2.5, the default value has been changed from 8 to 0.
+
+#### clear_transaction_task_worker_count
+
+- **Default**: 1
+- **Unit**: N/A
+- **Description**: The number of threads used for clearing transaction.
+
+#### alter_tablet_worker_count
+
+- **Default**: 3
+- **Unit**: N/A
+- **Description**: The number of threads used for schema change.
+
+#### clone_worker_count
+
+- **Default**: 3
+- **Unit**: N/A
+- **Description**: The number of threads used for clone.
+
+#### storage_medium_migrate_count
+
+- **Default**: 1
+- **Unit**: N/A
+- **Description**: The number of threads used for storage medium migration (from SATA to SSD).
+
+#### check_consistency_worker_count
+
+- **Default**: 1
+- **Unit**: N/A
+- **Description**: The number of threads used for checking the consistency of tablets.
+
+#### sys_log_dir
+
+- **Default**: ${STARROCKS_HOME}/log
+- **Unit**: N/A
+- **Description**: The directory that stores system logs (including INFO, WARNING, ERROR, and FATAL).
+
+#### user_function_dir
+
+- **Default**: ${STARROCKS_HOME}/lib/udf
+- **Unit**: N/A
+- **Description**: The directory used to store User-defined Functions (UDFs).
+
+#### small_file_dir
+
+- **Default**: ${STARROCKS_HOME}/lib/small_file
+- **Unit**: N/A
+- **Description**: The directory used to store the files downloaded by the file manager.
+
+#### sys_log_level
+
+- **Default**: INFO
+- **Unit**: N/A
+- **Description**: The severity levels into which system log entries are classified. Valid values: INFO, WARN, ERROR, and FATAL.
+
+#### sys_log_roll_mode
+
+- **Default**: SIZE-MB-1024
+- **Unit**: N/A
+- **Description**: The mode in which system logs are segmented into log rolls. Valid values include TIME-DAY, TIME-HOUR, and SIZE-MB-<size>. The default value indicates that logs are segmented into rolls, each of which is 1 GB.
+
+#### sys_log_roll_num
+
+- **Default**: 10
+- **Unit**: N/A
+- **Description**: The number of log rolls to reserve.
+
+#### sys_log_verbose_modules
+
+- **Default**: Empty string
+- **Unit**: N/A
+- **Description**: The module of the logs to be printed. For example, if you set this configuration item to OLAP, StarRocks only prints the logs of the OLAP module. Valid values are namespaces in BE, including starrocks, starrocks::debug, starrocks::fs, starrocks::io, starrocks::lake, starrocks::pipeline, starrocks::query_cache, starrocks::stream, and starrocks::workgroup.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/63614deb-7f22-493a-972b-01c7e338a6b2)
It would be better to wrap all these in code format.